README.md

```diff
@@ -28,6 +28,18 @@ bundle exec rackup
 You would then add the ingester as a Grafana webhook channel like so;
 ![Grafana config](https://i.imgur.com/Cu4m8Ew.png)
 
+## Docker
+
+Create a proper configuration file from the provided `config.yml.example`, then build the image:
```

Comment:
I'd suggest moving the configuration file info down below the build step, it's only interesting at the point of execution after all.

Reply:
Makes sense, thanks.